unfoldingWord/translationCore
# Improved door43 project search actions to return more results

#### This pull request addresses:

- Improved door43 project search actions to return more results


#### How to test this pull request:
- Previous search results for search criteria (**Language Code:** hi and **Book:** Titus)
![image](https://user-images.githubusercontent.com/8171759/32410123-51d31430-c187-11e7-982b-2394c24705d6.png)

-New search results based on the search criteria in the picture should return the projects in the picture above and the project in the picture below and more.

![image](https://user-images.githubusercontent.com/8171759/32410135-7fce1006-c187-11e7-8e8d-7543a8e2c3e6.png)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3132)
